deployment/updates/scripts/add_permissions/2017_05_15_add_poll_service.php

```diff
@@ -0,0 +1,9 @@
+<?php
+/**
+ * @package deployment
+ * @subpackage lynx.roles_and_permissions
+ */
+
+$script = realpath(dirname(__FILE__) . '/../../../../') . '/alpha/scripts/utils/permissions/addPermissionsAndItems.php';

```

Comment:
Needed if we want only registered users to create polls and get the votes as it is today